test/active_reporting/report_test.rb

```diff
@@ -32,6 +32,29 @@ def test_report_runs_with_an_aggregate_other_than_count
     assert data.all? { |r| r.key?('a_metric') }
   end
 
+  def test_report_runs_with_a_year_grouping
+    if ENV['DB'] == 'pg'
```

Comment:
Would this not work with MySQL as well?

Reply:
I did a helper function in the other pr, I can change here after merge the other.